webui/src/pages/bug/CommentForm.tsx

```diff
@@ -89,6 +92,7 @@ function CommentForm({ bugId }: Props) {
           onChange={(comment: string) => setIssueComment(comment)}
         />
         <div className={classes.actions}>
+          <CloseBugButton bug={bug} disabled={issueComment.length > 0} />
```

Comment:
that's a weird `disabled` condition. Could you explain why?

Reply:
Sure! The CloseButton is only enabled when there is no comment typed.

When You enter in bug details page, Close Button is enabled (because the textfield for comments is empty) and Comment button is disabled. When You type the first key in the textfield for comments, Close button become disabled and Comment button become enabled.

Github shows a "Close and Comment" button when the user starts to type a comment but our mutation for closeBug don´t accept a comment as argument. 😉 

input CloseBugInput {
    """A unique identifier for the client performing the mutation."""
    clientMutationId: String
    """"The name of the repository. If not set, the default repository is used."""
    repoRef: String
    """The bug ID's prefix."""
    prefix: String!
}

Reply:
I can change the inline expression with a named function called hasNoComment() if You prefer.

Reply:
Hoooo, that's the textfield! I thought it was the number of comment on a bug.

> but our mutation for closeBug don´t accept a comment as argument.

You actually can do multiple mutation in one query in GraphQL, but I'm not sure how that translate in the tsx code: https://hasura.io/docs/1.0/graphql/core/mutations/multiple-mutations.html

Reply:
Let's try to do that? It will be useful for some other things anyway.

Reply:
No problem! I can try it. I will create an issue for that.

Can You accept this PR? I have one more commit to put an end to the confusion I created. 🙏

Reply:
:+1: